builds/assets/lang/enAU.json

```diff
@@ -329,7 +329,8 @@
   "flythroughFPS": "FPS",
   "flythroughDefaultName": "Camera Path",
   "flythroughExport": "Export",
-  "flythroughExportStop": "Stop",
+  "flythroughExporting": "Exporting Flythrough...",
+  "flythroughExportCancel": "Cancel",
```

Comment:
Instead of adding new keys (which will trigger requiring new translations at the last second) use these:
  - `sceneExplorerCancelButton` and;
  - `sceneExplorerExportRunning`